docs/aws-openedx/aws-deployment.md

```diff
@@ -0,0 +1,36 @@
+# Open edX AWS Production Deployment and Upgrades
+
+In contrast to OCIM updates, which are automated through a control panel, AWS updates are more manual and we have to actually log in to the AWS panel and SSH to the instance.
+Apart from this, the main part of the deployment process is an `ansible` playbook which does all the provisioning automatically.
+
```

Comment:
The audience for this documentation repository is the public, and so they don't necessarily know about OCIM.  Please remove these references, and update this intro to suit an AWS-only universe.